docs/sources/next/using-k6/metrics/_index.md

```diff
@@ -111,8 +111,4 @@ For details of all metrics, refer to the [Metrics reference](https://grafana.com
 
 Metric names must comply with OpenTelemetry and [Prometheus limitations](https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels). The character limit is the same limit k6 had before restricting the character set down.
 
-That means they must not start with a number and a metric name can be 1 to 128 symbols of:
-
-1. any Unicode Letters
-2. any Unicode Number
-3. `_` (an underscore)
+That means metrics names must only include up to 128 ASCII letters, numbers, or underscores and start with a letter or an underscore.
```

Comment:
```suggestion
That means metrics names must:

- Start with a letter or an underscore.
- Include up to 128 symbols (ASCII letters, numbers, or underscores).
```

I like keeping it as a list, I think it makes it easier to parse. What do you think? 🤔

Reply:
TBH, in that particular case, for me, a single sentence seems more straightforward than a list of two items. 

On the other hand, I'm not firm about that, so if you think that the list is better for a reader, I'll trust you and will be happy to go with the list :relaxed:

Reply:
@mstoykov could also bring an input :+1:

Reply:
I am fine with both, but I would reverse the order of the bullet points, as `letter` in the first is not specified to be ASCII one, so IMO makes it harder to grasp.

Reply:
I reversed the order of the bullet points in my suggestion. :)

It's a small thing, but I do think, generally, lists are easier to grasp for readers, especially when you're talking about conditions.